doc/Type/Cool.pod6

```diff
@@ -1163,12 +1163,18 @@ L<Regex|/type/Regex> as a list of strings.
 
     multi method contains(Cool:D: Str(Cool) $needle, Cool $start? --> Bool:D)
 
+Coerces the first two arguments (in method form, also counting the invocant)
+to L<Str|/type/Str>, and searches for C<$needle> in the string starting from C<$start>.
 Returns C<True> if the invocant contains the C<$needle> at any position within
 the string. If $start is provided skip as many characters.
 
     say "Hello, World".contains('hello');      # OUTPUT: «False␤»
```

Comment:
I'm wondering what the output for `.contains('Hello")` would be.
(Sorry if that is out of scope of the PR.)